cli/src/config.rs

```diff
@@ -23,7 +23,7 @@ fn read_config_file(path: &Path) -> Result<HashMap<String, Value>, Box<dyn std::
 /// Only the options that do not appear in the command line will be appended.
 fn extend_cli_args(
     cli_args: Vec<String>,
-    path: &std::path::Path,
+    optional_path: Option<&std::path::Path>,
```

Comment:
```suggestion
    optional_path: Option<&Path>,
```

Reply:
the destructuring `if-let` assignment below take the `path` identifier, though it's also work.  Shall I rename the variable?